app/src/app/cities-list/cities-list.component.html

```diff
@@ -0,0 +1,84 @@
+<app-wrapper>
+  <div class="search-container">
+    <form [formGroup]="searchForm" (ngSubmit)="onSearchClick()">
+      <mat-form-field class="form-field full-width" appearance="outline">
+        <input
+          placeholder="Search for a city"
+          type="text"
+          matInput
+          formControlName="name"
+        />
+      </mat-form-field>
+      <div class="search-buttons-container">
+        <button
+          class="reset-button"
+          type="reset"
+          (click)="resetSearch()"
+          mat-raised-button
+        >
+          Reset
+        </button>
+        <button class="search-button" type="submit" mat-raised-button>
+          Search
+        </button>
+      </div>
+    </form>
+  </div>
+
+  <mat-paginator
```

Comment:
Just to confirm my suspicions here - are we adding two paginators so that the user can move between the results from both places (top and bottom)?

Reply:
Correct!